modules/policy/engine/src/test/java/org/eclipse/sw360/antenna/policy/workflow/processors/testdata/EPLvsGPLRule.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.workflow.processors.testdata;
+
+import org.eclipse.sw360.antenna.policy.engine.*;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.eclipse.sw360.antenna.policy.engine.RuleUtils.artifactAppliesToRule;
+import static org.eclipse.sw360.antenna.policy.engine.RuleUtils.artifactRaisesPolicyViolation;
+
+public class EPLvsGPLRule implements CompareArtifactRule {
+    private final Ruleset ruleset;
+
+    public EPLvsGPLRule(final Ruleset ruleset) {
+        this.ruleset = ruleset;
+    }
+
+    @Override
+    public Optional<PolicyViolation> evaluate(final ThirdPartyArtifact leftArtifact,
+            final ThirdPartyArtifact rightArtifact) {
+        if (leftArtifact.hasAtLeastOneLicenseOf(Arrays.asList("EPL-1.0"))
+                && rightArtifact.hasAtLeastOneLicenseOf(Arrays.asList("GPL-2.0-or-later"))) {
```

Comment:
Just a though: Is there a risk that the deprecated id for this license, i.e. `GPL-2.0+`, is being used? If so, it should probably be listed here, too.

Reply:
This is a test version, so I only expect what I give in the testdata :-). But in general, I thought about this when implementing the rule. My conclusion was, that this should not be the case, since the data in entered by the OSO office, so we have to teach Sheriff, which identifiers to use. In addition, I think of a rule, that checks for proper usage of SPDX identifiers, resp. gives warnings on unknown identifiers